dot/sync/message_integeration_test.go

```diff
@@ -32,7 +35,7 @@ func addTestBlocksToState(t *testing.T, depth uint, blockState BlockState) {
 				ParentHash: previousHash,
 				Number:     previousNum + i,
 				StateRoot:  trie.EmptyHash,
-				Digest:     digest,
+				Digest:     types.NewDigest(),
```

Comment:
Why was this changed from `digest` (with that `prd` added)?

If there is reason, maybe remove the `digest` variable since it's not used I think.

Reply:
Not sure why this was changed, changed it back and confirmed test that use this are working.